### PR TITLE
Add note to expect changes to tripartite syntax and semantics

### PR DESCRIPTION
**Pull request by heplesser**
This PR add a note to documentation to alert users about upcoming changes in the syntax and semantics for tripartite connectivity.

### CODE REVIEW

**Review by HanjiaJiang — APPROVED** [review]
This looks good. Thank you!

File: doc/htmldoc/synapses/connection_management.rst [hunk 1]
```diff
@@ -314,6 +314,12 @@ specification to connections from ``pre`` to ``third`` nodes and
 the ``'third_out'`` specification to connections from ``third`` to
 ``post`` nodes.
 
+.. admonition:
+
+   Tripartite connectivity is a new feature. Please expect some adjustments to
+   the syntax and semantics for for tripartite connectivity in the next NEST releases.
```
Comment (jessica-mitchell) [inline doc/htmldoc/synapses/connection_management.rst:320]:
```suggestion
   the syntax and semantics for the tripartite connectivity in the next NEST releases.
```

File: pynest/nest/lib/hl_api_connections.py [hunk 2]
```diff
@@ -366,6 +366,12 @@ def TripartiteConnect(pre, post, third, conn_spec, syn_specs=None):
      - any parameters specific to the selected synapse model.
 
 
+    .. admonition:
+
+       Tripartite connectivity is a new feature. Please expect some adjustments to
+       the syntax and semantics for for tripartite connectivity in the next NEST releases.
```
Comment (jessica-mitchell) [inline pynest/nest/lib/hl_api_connections.py:372]:
```suggestion
       the syntax and semantics for the tripartite connectivity in the next NEST releases.
```

**Review by jessica-mitchell — APPROVED** [review]
@heplesser There are a couple of typos to fix, but I approve

**Comment by heplesser** [comment]
@jessica-mitchell I committed your suggestions so feel free to merge once the tests are through.

**Pull request merged**